### PR TITLE
Remove Doctrine mappings YML from documentation

### PR DESCRIPTION
**Pull request by cosminsandu**
https://github.com/symfony/symfony-docs/issues/20050

### CODE REVIEW

File: reference/configuration/doctrine.rst [hunk 1]
```diff
@@ -269,9 +269,14 @@ you can control. The following configuration options exist for a mapping:
 ........
 
 One of ``annotation`` (for PHP annotations; it's the default value),
-``attribute`` (for PHP attributes), ``xml``, ``yml``, ``php`` or
+``attribute`` (for PHP attributes), ``xml``, ``php`` or
 ``staticphp``. This specifies which type of metadata type your mapping uses.
 
+.. deprecated:: ORM 3.0
```
Comment (xabbuh) [inline reference/configuration/doctrine.rst:275]: @cosminsandu I think you should be able to add an entry to the list in `.doctor-rst.yaml`

Comment (xabbuh) [inline reference/configuration/doctrine.rst]:
```suggestion
.. deprecated:: 3.0
```

@cosminsandu I think you should be able to add an entry to the list in `.doctor-rst.yaml`

**Comment by cosminsandu** [comment]
@xabbuh Now I get:
```
Error: You are not allowed to use version "3.0". Only major version "5" is allowed.
Error: Please only provide ".. deprecated::" if the version is greater/equal "5.0"
```

**Comment by xabbuh** [comment]
I think you should be able to add an entry to the list in `.doctor-rst.yaml`.

**Comment by javiereguiluz** [comment]
Thanks Cosmin ... and congrats on your first Symfony Docs contribution 🎉

**Pull request merged**